sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -52,6 +56,7 @@
 from apache_beam.transforms.window import GlobalWindows
 from apache_beam.utils import proto_utils
 
+standard_library.install_aliases()
```

Comment:
Do we want to call this before we do the import of "queue" above?  It looks like this only works because of an "accident" in that this module is imported after someone else already called `install_aliases()`.